target/linux/ramips/image/mt7621.mk

```diff
@@ -328,6 +328,15 @@ define Device/wndr3700v5
 endef
 TARGET_DEVICES += wndr3700v5
 
+define Device/wr1200js
+  DTS := WR1200JS
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
```

Comment:
`ralink_default_fw_size_16M` doesn't really fit here. ralink_default_fw_size_16M == 16121856 bytes == 0xf60000 bytes. but your firmware partition has a size of 0xfb0000.

Use `IMAGE_SIZE := 16064k` instead.